frontend/packages/llama-stack-modular-ui/bff/internal/constants/keys.go

```diff
@@ -11,5 +11,6 @@ const (
 	TraceLoggerKey contextKey = "TraceLoggerKey"
 
 	// The following keys are used to store the user access token in the context
-	RequestIdentityKey contextKey = "requestIdentityKey"
+	RequestIdentityKey          contextKey = "requestIdentityKey"
+	NamespaceHeaderParameterKey contextKey = "namespace"
```

Comment:
Can we rename it to NamespaceQueryParameterKey ; it will be confusing in the long term.